src/hooks/use-element-should-close.js

```diff
@@ -76,7 +78,9 @@ export function useElementShouldClose(closeableEl, isOpen, handleClose) {
       document.body,
       ['mousedown', 'click'],
       event => {
-        if (!closeableEl.current.contains(/** @type {Node} */ (event.target))) {
+        if (
+          !closeableEl.current?.contains(/** @type {Node} */ (event.target))
```

Comment:
Please, @lyzadanger could you double check the intended behaviour here?

Reply:
Unrelated: listening for just `mousedown` could be enough. I believe some browsers in mobile device have some issues with tap event not generating `mousedown` events. A little bit of investigation would be required.